playbooks/installation/index.adoc

```diff
@@ -127,41 +127,32 @@ More information on Pods & Services can be found in the link:https://docs.opensh
+In an HA cluster there should also be two DNS names for the Load Balanced IP address that points to the 3 master servers for access to the API, CLI and Console services. One of these names is the *public* name that users will use to log into the cluster. The other is an _internal_ name that will be used by internal components within the cluster to talk back to the master. These values should also resolve, and will be placed in the ansible Hosts file for the variables.
 
-Public Master URL
+==== Public Master Hostname
 
-This will be the url that external users and/or tools will use to login to the OpenShift cluster
+*Ansible Inventory Variable:* `openshift_master_cluster_public_hostname=console.c1-ocp.myorg.com`
 
-Ansible Host file variable:
+This will be the hostname that external users and/or tools will use to login to the OpenShift cluster
 
-*Master Public Hostname Variable :* `openshift_master_cluster_public_hostname=console.c1-ocp.myorg.com`
+==== Internal Master Hostname
 
-Replace the value in your host file
+*Ansible Inventory Variable:* `openshift_master_cluster_hostname=console-int.c1-ocp.myorg.com`
 
+This will be the hostname that external users and/or tools will use to login to the OpenShift cluster
 
-Master URL
+==== Wildcard DNS entry for Infrastructure(Router) nodes
 
-This will be the url that external users and/or tools will use to login to the OpenShift cluster
+*Ansible Inventory Variable:* `openshift_master_default_subdomain=apps.c1-ocp.myorg.com`
 
-*Master Hostname Variable:*`openshift_master_cluster_hostname=console.c1-ocp.myorg.com`
-
-Wildcard DNS entry for Infrastructure(Router) nodes
-
-In addition to the hostnames for the master console and API, A wildcard DNS entry needs to exist under a unique subdomain (i.e. `*.cloudapps.c1-ocp.myorg.com`) that resolves to either the IP addresses (an A record) or the hostnames (a CNAME record) of the three Infrastructure Nodes.   This entry allows new routes to be automatically routable to the cluster under the subdomain such as mynewapp.cloudapps.c1-ocp.myorg.com.   Alternatively, every exposed route would require the entry to be created in order to route it to the OpenShift cluster.
-
-----
-# default subdomain to use for exposed routes
-openshift_master_default_subdomain=cloudapps.c1-ocp.myorg.com
-----
+In addition to the hostnames for the master console and API, A wildcard DNS entry needs to exist under a unique subdomain (i.e. `*.apps.c1-ocp.myorg.com`) that resolves to either the IP addresses (an A record) or the hostnames (a CNAME record) of the three Infrastructure Nodes.   This entry allows new routes to be automatically routable to the cluster under the subdomain such as mynewapp.apps.c1-ocp.myorg.com.   Alternatively, every exposed route would require the entry to be created in order to route it to the OpenShift cluster.
```

Comment:
Maybe a note here pointing to the event-controller as an option (with the proper plugin enabled/added)?